core/pom.xml

```diff
@@ -196,21 +159,11 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <useSystemClassLoader>true</useSystemClassLoader>
-                    <useManifestOnlyJar>true</useManifestOnlyJar>
+                    <argLine>${additionalRuntimeArgLine}</argLine>
+                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
                     <systemPropertyVariables>
                         <jberet.tmp.dir>${project.build.directory}${file.separator}tmp</jberet.tmp.dir>
                     </systemPropertyVariables>
-                    <!-- ensure we don't inherit a byteman jar from any env settings -->
-                    <environmentVariables>
-                        <BYTEMAN_HOME></BYTEMAN_HOME>
-                    </environmentVariables>
-                    <systemProperties>
-                        <property>
-                            <name>org.jboss.byteman.home</name>
-                            <value></value>
-                        </property>
-                    </systemProperties>
```

Comment:
This was the configuration example provided by byteman in [BMUnit: How Do I Configure Injection Into JUnit With Maven?](https://developer.jboss.org/wiki/BMUnitUsingBytemanWithJUnitOrTestNGFromMavenAndAnt#how_do_i_configure_injection_into_junit_with_maven) Would this deletion cause problem if user have already `BYTEMAN_HOME` configured in environment variables?